samples/peripheral/802154_phy_test/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.10)
```

Comment:
It's `3.20` now.

Reply:
Actually, hold on with this change, there's some CI issue with higher CMake requirement.